.eslintrc.js

```diff
@@ -1,10 +1,11 @@
 module.exports = {
-  extends: '@mate-academy/eslint-config',
+  extends: "@mate-academy/eslint-config",
   env: {
-    jest: true
+    jest: true,
   },
   rules: {
-    'no-proto': 0
+    "no-proto": 0,
+    quotes: "off",
```

Comment:
The `quotes` rule is set to "off", which means ESLint will not enforce any specific quote style. If this is intentional, you can leave it as is. However, if you want to enforce a specific quote style, consider setting it to `"single"` or `"double"`.